eXo/en.lproj/InfoPlist.strings

```diff
@@ -16,4 +16,4 @@
 // 02110-1301 USA, or see the FSF site: http://www.fsf.org.
 
 
-"Connect Community" = "Community";
\ No newline at end of file
+"Connect Community" = "Community";
```

Comment:
I want to use a pattern to define the keys. I will define it in the tech spec. Something like `Shortcut.ConnectCommunity` seems better.